lib/apiservers/service/restapi/handlers/common.go

```diff
@@ -136,7 +136,7 @@ func getVCHConfig(op trace.Operation, d *data.Data) (*config.VirtualContainerHos
 	executor := management.NewDispatcher(validator.Context, validator.Session, nil, false)
 	vch, err := executor.NewVCHFromID(d.ID)
 	if err != nil {
-		return nil, util.NewError(500, fmt.Sprintf("failed to create VCH %s: %s", d.ID, err))
+		return nil, util.NewError(500, fmt.Sprintf("Unable to find VCH %s: %s", d.ID, err))
```

Comment:
Sorry I missed this on my first pass: should this be a 404?

Reply:
Yea, 500 isn't the best choice. Since the ID provided by the API client should be an existing one, is 400 bad request more suitable? 
This is copied from `common.go`, if this applies for all API handlers, I'll update `common.go` as well

Reply:
Oh actually 404 is better. it's "VCH not found", not "invalid VCH ID syntax"